### DISCUSSION TITLE
[Q&A]: Blinking when switching pages

### DISCUSSION THREAD

**Discussion by nayounsang** [wojtekmaj/react-pdf#1892] [Q&A] [answered]
hello. I am developing using React 18.
My project shows one pdf page on the screen.
```
<Document/>
  <Page pageIndex={pageIndex}/>
<Document/>
```
Users switch pageIndex to call setPageIndex() in `[pageIndex, setPageIndex] = useState<number>(0);`.
In the process of changing the page, elements are rebuilt and users see a white screen while doing so.
This is visually unpleasant.
How can I solve this? What methods are there?
I tried adjusting the zIndex of the previous page higher while loading to create a smooth transition, but that didn't work.

**Comment by wojtekmaj** [comment, marked as answer]
Duplicate of #1888 
Duplicate of #418
